src/account.js

```diff
@@ -27,35 +28,28 @@ const switchToLoggedIn = (message) => {
           location.reload());
       } else {
         username = response.entry.data;
-        skyid.getProfile((response2) => {
-          /*response2 = JSON.parse(response2);*/
-          pubkey = response2.dapps["Hacker Paste"].publicKey;
-          byId("username").textContent = username;
-          byId("button-username").setAttribute('aria-label', 'View My Pastes');
-          byId("button-username").setAttribute('data-microtip-size', 'fit');
-          skyid.getJSON('hackerpaste:my-pastes', (response3) => {
-            if (response3 !== null) {
-              myPastes = decryptObject(response3, skyid.seed);
-              deleteClickListener("button-username", startSkyIDSession);
-              clickListener("button-username", loadMyPastes);
-              MicroModal.close('app-modal');
-            } else {
-              let noteToSelf = getPubkeyBasedRetrievalString(
-                pubkey) + generateDocKey();
-              let defaultContent =
-                {documents:[{label:"Note to Self",docID:noteToSelf}]};
-              defaultContent = encryptObject(defaultContent, skyid.seed);
-              skyid.setJSON('hackerpaste:my-pastes',
-                defaultContent, () => location.reload());
-            }
-          });
+        byId("username").textContent = username;
+        byId("button-username").setAttribute('aria-label', 'View My Pastes');
+        byId("button-username").setAttribute('data-microtip-size', 'fit');
+        skyid.getJSON('hackerpaste:my-pastes', (response3) => {
+          if (response3["entry"] !== null) {
+            //myPastes = decryptObject(response3, skyid.seed);
+            let myPastes = response3["entry"]
+            deleteClickListener("button-username", startSkyIDSession);
+            clickListener("button-username", loadMyPastes);
+            MicroModal.close('app-modal');
+          } else {
+            let defaultContent = {documents:[{label: "Note to Self", docID:Math.random()*1000}]}
```

Comment:
I do not recommend using Math.random for generating randomness since random number generators are used in Hacker Paste in a security-sensitive context (i.e. decryption keys).